src/cli.rs

```diff
@@ -609,7 +588,7 @@ impl Cli {
         static ARG_STORAGE: OnceCell<Vec<String>> = OnceCell::new();
```

Comment:
I think we can get rid of this whole kludge now and just pass `String`s into `Arg::new()`